package.json

```diff
@@ -78,55 +78,62 @@
     "test:ci:integration": "yarn start-test start:all 8080 cy:run"
   },
   "devDependencies": {
-    "@angular-builders/custom-webpack": "12.1.2",
-    "@angular-devkit/build-angular": "12.2.7",
-    "@angular-devkit/core": "12.2.7",
-    "@angular-devkit/schematics": "12.2.7",
-    "@angular/animations": "12.2.7",
-    "@angular/cli": "12.2.7",
-    "@angular/common": "12.2.7",
-    "@angular/compiler": "12.2.7",
-    "@angular/compiler-cli": "12.2.7",
-    "@angular/core": "12.2.7",
-    "@angular/elements": "12.2.7",
-    "@angular/forms": "12.2.7",
-    "@angular/language-service": "12.2.7",
-    "@angular/platform-browser": "12.2.7",
-    "@angular/platform-browser-dynamic": "12.2.7",
-    "@angular/router": "12.2.7",
-    "@commitlint/cli": "^8.3.5",
-    "@commitlint/config-conventional": "^8.3.4",
-    "@nrwl/angular": "12.9.0",
-    "@nrwl/cli": "12.9.0",
-    "@nrwl/nx-cloud": "12.3.13",
-    "@nrwl/workspace": "12.9.0",
-    "@types/jest": "^26.0.10",
+    "@angular-builders/custom-webpack": "13.0.0",
```

Comment:
Changes to these devDependencies also probably will result in build artifacts that don't work with Angular 12 - another reason to publish this as single-spa-angular@6